src/fillTank.test.js

```diff
@@ -1,11 +1,98 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+   const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared ', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should calculate full tank If the amount is not given', () => {
```

Comment:
The test description mentions calculating a full tank if the amount is not given, but the test case is passing an amount of 30. According to the task description, if the amount is not given, the function should fill the tank to its maximum capacity. Consider adjusting the test case to not pass an amount if you want to test this scenario.

Reply:
30 is fuel price not amount 
Function fillTank(customer, fuelPrice, amount = Infinity)